catalog/app/containers/Bucket/Queries/requests/athena.ts

```diff
@@ -103,12 +103,19 @@ async function fetchWorkgroup({
   athena,
   workgroup,
 }: WorkgroupArgs): Promise<Workgroup | null> {
-  const workgroupOutput = await athena.getWorkGroup({ WorkGroup: workgroup }).promise()
-  return workgroupOutput?.WorkGroup?.Configuration?.ResultConfiguration?.OutputLocation &&
-    workgroupOutput?.WorkGroup?.State === 'ENABLED' &&
-    workgroupOutput?.WorkGroup?.Name
-    ? workgroupOutput.WorkGroup.Name
-    : null
+  try {
+    const workgroupOutput = await athena.getWorkGroup({ WorkGroup: workgroup }).promise()
+    if (
+      workgroupOutput?.WorkGroup?.Configuration?.ResultConfiguration?.OutputLocation &&
+      workgroupOutput?.WorkGroup?.State === 'ENABLED' &&
+      workgroupOutput?.WorkGroup?.Name
+    ) {
+      return workgroupOutput.WorkGroup.Name
+    }
+    return null
+  } catch (error) {
```

Comment:
it would be nice to log this error at least, or better yet, catch only specific type of errors (check on `.code`?) and re-throw unexpected ones

Reply:
Check for HTTP `code` is a good one. I'll try to do it next. I want to merge it as is just to make staging work sooner

Reply:
i meant AWS error code, but you get the idea